testing/rtrlib/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: Christian Franke <nobody@nowhere.ws>
+pkgname=rtrlib
+pkgver=0.6.3
+pkgrel=0
+pkgdesc="An open-source C implementation of the RPKI/Router Protocol client"
+url="https://github.com/rtrlib/rtrlib"
+arch="all"
+license="MIT"
+depends=""
```

Comment:
no reason to keep empty lines

Reply:
Good point, thank you for pointing it out. :pray: